src/Pos/Block/Network/Types.hs

```diff
@@ -72,3 +73,11 @@ newtype MsgBlock ssc =
     deriving (Generic, Show)
 
 deriving instance (Ssc ssc, Eq (SscPayload ssc)) => Eq (MsgBlock ssc)
+
+-- | 'Subscribe' message
+--
+-- This can be used by behind-NAT nodes to subscribe to the 'OutboundQueue'
+-- of a relay node. The node will remain subscribed for the duration of the
+-- conversation.
```

Comment:
Might be worth noting in the docs here that it's not just behind NAT, will also be used for kademlia (including core in traditional mode)